lib/wasi/src/wapm/mod.rs

```diff
@@ -503,3 +438,190 @@ fn count_file_system(fs: &dyn FileSystem, path: &Path) -> u64 {
 
     total
 }
+
+#[cfg(test)]
+mod tests {
+    use std::collections::BTreeMap;
+
+    use super::*;
+
+    const PYTHON: &[u8] = include_bytes!("../../../c-api/examples/assets/python-0.1.0.wasmer");
```

Comment:
Can we add a test for one of the currently used static webserver webcs, to make sure our current hacky solution of falling back to the first atom in a dependency continues to work?